### DISCUSSION TITLE
Error FFmpegReturnCodeError: FFmpeg did not return 0

### DISCUSSION THREAD

**Discussion by Kermitofx** [MarshalX/tgcalls#91] [Q&A] [answered]
FFmpegReturnCodeError: FFmpeg did not return 0

**Comment by rojvv** [comment, marked as answer]
First of all, this is not related to this library in any matters.

It's an error [raised](https://github.com/callsmusic/callsmusic/blob/08ead6604cd591f97d61ecc6c45f6ac6a01cdbb1/callsmusic/converter/converter.py#L27) by Calls Music, you can ask in its [chat](https://t.me/callsmusicchat) for help.

And for the next time, when you start a discussion or issue, please make sure you're doing it in the right respository/place.

Thanks!
